### PR TITLE
Omit LastDeployed when it's empty

### PR DESCRIPTION
**Pull request by tomahawk28**
Toward giantswarm/giantswarm#10644

Removing `lastDeployed` from the required variables. 

## Checklist

- [ ] Consider SIG UX feedback.
- [x] Update changelog in CHANGELOG.md.
- [x] If adding a new type, ensure that it is added to the [CRD unit tests](https://github.com/giantswarm/apiextensions/blob/d78aba91d578d56ef49f58e336035d35edc4e1b0/pkg/crd/crd_test.go#L9).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- No longer required to have `lastDeployed` in App, Chart CRs.
```
Comment (rossf7) [inline CHANGELOG.md:14]:
```suggestion
- Make .status.release.lastDeployed of app and chart CRs optional.
```

What about this? I stole it from `Make .status.kvm.nodeIndexes of KVMConfig optional.` 😄

**Review by rossf7 — APPROVED** [review]
LGTM just a suggestion on the changelog

File: CHANGELOG.md (hunk 1 above)
Comment (tomahawk28) [reply CHANGELOG.md:14]: Nice, Let's use this.

**Comment by marians** [comment]
ACK

**Pull request merged**